packages/server/src/models/user.entity.ts

```diff
@@ -17,9 +20,9 @@ export default class User {
   @Column()
   public status: string;
 
-  @Column()
-  public facebookId: string;
+  @Column({ nullable: true })
+  public facebookId?: string;
 
-  @Column()
-  public googleId: string;
+  @Column({ nullable: true })
+  public googleId?: string;
```

Comment:
Maybe we should move facebookId and googleId to a separate entity?
Just like with email confirmation and password reset entities.  Just my thoughts.

Reply:
As for now we have just facebookId, I believe not only it but other fields like it are to be moved into a separate entity but that feels rather like a generic improvement of out DB structure